include/tcpdf_colors.php

```diff
@@ -449,7 +449,7 @@ public static function _JScolor($color) {
 		}
 		if (!in_array($color, self::$jscolor)) {
 			// default transparent color
-			$color = $jscolor[0];
+			$color = self::$jscolor[0];
```

Comment:
Note this code has several bugs and it is way behind:
https://github.com/tecnickcom/tc-lib-barcode
It would be better to switch to the new library.

Reply:
This is pure typo fix, these is no `jscolor` prop